source/manual/incident-reports.html.md

```diff
@@ -60,30 +60,34 @@ Time to fix (time from when incident declared to marked as resolved)|
 ### Time:
 ### Attendance:
 
-### Root cause
+### Root cause analysis
 
 Use this section to summarise the root cause of the incident. A draft root cause should be included, and discussed and agreed upon during the incident review. This root cause should be written for a non-technical audience.
  
 Please also include the root cause category:
-
 * Unexpected effect of code change
-* Infrastructure failure (disk space, logging etc)
-* Provider failure (hardware, network failure etc
-* Security
+* Configuration fault (configuration change, pre-existing fault, etc)
+* Capacity issue (insufficient capacity to meet demand, implementation issue, etc)
+* Provider / third-party infrastructure failure (hardware, network failure, etc)
+* Workflow issue (usability issue, misoperation, etc)
+* Software design / architecture problem (i.e. worked as designed but led to a problem)
+* Other (please specify)
 
 ...
 ### Actions
 
-Use this section to assign actions to individuals (not teams). These are actions to be taken to fix the root cause of the issue or for preventative measures.
+Use this section to assign actions to individuals (not teams). These are actions to be taken to fix the root cause of the issue, for preventative measures and for any other improvements.
 
 * ...
+
 ### Standing actions
 
```

Comment:
Should an extra bullet be added to the list below -> `Update your incident log with incident title, priority, dates/times and a link to this report.` - although I don't actually know what an incident log is in this context ?